lib/service_check.pm

```diff
@@ -263,7 +263,8 @@ sub install_services {
     }
     # On ppc64le, sometime the console font will be distorted into pseudo graphics characters.
     # we need to reset the console font. As it impacted all the console services, added this command to bashrc file
-    systemctl('restart systemd-vconsole-setup.service') if is_ppc64le;
+    assert_script_run('echo /usr/lib/systemd/systemd-vconsole-setup >> /etc/bash.bashrc.local') if is_ppc64le;
+    assert_script_run '. /etc/bash.bashrc.local';
```

Comment:
what's the enable status of systemd-vconsole-setup.service? It should work is service is enabled after system reboot, otherwise it should be a bug.

Reply:
OK, I will try

Reply:
Cannot work will report one bug